var/spack/repos/spack_repo/builtin/packages/paraview/package.py

```diff
@@ -75,7 +75,8 @@ class Paraview(CMakePackage, CudaPackage, ROCmPackage):
     variant("fortran", default=False, description="Enable Fortran support")
     variant("mpi", default=True, description="Enable MPI support")
     variant("qt", default=False, description="Enable Qt (gui) support")
-    variant("opengl2", default=True, description="Enable OpenGL2 backend")
+    variant("opengl2", default=True, description="Enable OpenGL2 backend", when="@5:5")
+    variant("use_x", default=True, description="Enable OpenGL2 backend")
```

Comment:
Why not simply `x`? That seems more standard in our packages.

Reply:
I see, I am not a huge fan of single letter variants but if guess if there is a precedence already we can do that.